docs/developer-guides/merkle-root-construction.md

```diff
@@ -41,7 +76,7 @@ hashed to produce the merkle root.
          sha256(sha256("82501c1178fa0b222c1f3d474ec726b832013f0a532b44bb620cce8624a5feb1169e1e83e930853391bc6f35f605c6754cfead57cf8387639d3b4096c54f18f4".decode("hex")).digest()).digest().encode("hex_codec")
 -->
 
-![Example Merkle Tree Construction](/img/dev/en-merkle-tree-construction.svg)
+![Example Merkle Tree Construction](../img/core-blockchain-concepts/merkle-tree-construction.svg)
```

Comment:
`/img/core-blockchain-concepts/merkle-tree-construction.svg`

Reply:
Fixed

Reply:
Doesnt seem to be, I am still seeing a relative path beginning with `../`

Reply:
Ah, that was in the leftover Bitcoin template material. It looks like we need to get rid of that anyway, so have removed.